From cc031eea3f4a7a4633a34c8db0aa7762e00195a3 Mon Sep 17 00:00:00 2001
From: Rei Oliveira <Reinildes.JoseMateusOliveira@windriver.com>
Date: Fri, 14 Jun 2024 20:29:10 -0300
Subject: [PATCH] Remove oslotest.

This dependency is only used for testing and is not required for the
debian build or during runtime.
The reason for removal is that it's causing build dependency issues.

Signed-off-by: Rei Oliveira <Reinildes.JoseMateusOliveira@windriver.com>
---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 6e9b6b3..c445255 100644
--- a/debian/control
+++ b/debian/control
@@ -25,7 +25,6 @@ Build-Depends-Indep:
  python3-os-service-types,
  python3-oslo.config,
  python3-oslo.utils,
- python3-oslotest,
  python3-pep8,
  python3-requests,
  python3-requests-kerberos,
--
2.34.1